zebra-rpc/Cargo.toml

```diff
@@ -28,7 +28,7 @@ jsonrpc-derive = "18.0.0"
 jsonrpc-http-server = "18.0.0"
 # zebra-rpc needs the preserve_order feature in serde_json, which is a dependency of jsonrpc-core
 serde_json = { version = "1.0.81", features = ["preserve_order"] }
-indexmap = { version = "1.8.2", features = ["serde"] }
+indexmap = { version = "1.9.1", features = ["serde"] }
```

Comment:
Maybe we merge this one first? https://github.com/ZcashFoundation/zebra/pull/4676

Reply:
Should we block this PR on that PR?

Reply:
This PR will need a `@dependabot recreate` after PR #4676 merges